parachains/runtimes/collectives/collectives-polkadot/src/fellowship/migration.rs

```diff
@@ -0,0 +1,264 @@
+					// 2 writes to IdToIndex and IndexToId.
+					weight.saturating_accrue(T::DbWeight::get().writes(2));
+				}
+			}
+			// writes to MemberCount.
+			weight.saturating_accrue(T::DbWeight::get().writes((max_rank as u64) + 1));
+
+			weight
+		}
+
+		#[cfg(feature = "try-runtime")]
+		fn post_upgrade(_state: Vec<u8>) -> Result<(), &'static str> {
+			assert_eq!(MemberCount::<T, I>::get(0), 46, "invalid members count at rank 0.");
+			assert_eq!(MemberCount::<T, I>::get(1), 46, "invalid members count at rank 1.");
+			assert_eq!(MemberCount::<T, I>::get(2), 24, "invalid members count at rank 2.");
+			assert_eq!(MemberCount::<T, I>::get(3), 17, "invalid members count at rank 3.");
+			assert_eq!(MemberCount::<T, I>::get(4), 10, "invalid members count at rank 4.");
+			assert_eq!(MemberCount::<T, I>::get(5), 7, "invalid members count at rank 5.");
+			assert_eq!(MemberCount::<T, I>::get(6), 3, "invalid members count at rank 6.");
+			assert_eq!(MemberCount::<T, I>::get(7), 0, "invalid members count at rank 7.");
+			Ok(())
+		}
+	}
+}
+
+#[cfg(test)]
+pub mod tests {
+	use super::import_kusama_fellowship::FellowshipAddresses;
+	use crate::{FellowshipCollectiveInstance as Fellowship, Runtime, System};
+	use frame_support::traits::OnRuntimeUpgrade;
+	use pallet_ranked_collective::Rank;
+	use parachains_common::AccountId;
+	use sp_core::crypto::Ss58Codec;
+	use sp_runtime::AccountId32;
+
+	#[test]
+	fn check_fellowship_addresses() {
+		let fellowship_addresses = FellowshipAddresses::get();
+		let kusama_fellowship_ss58: [(Rank, _); 46] = [
+			(6, "16SDAKg9N6kKAbhgDyxBXdHEwpwHUHs2CNEiLNGeZV55qHna"), // proof https://kusama.subscan.io/extrinsic/16832707-4
+			(6, "J8ww78Qx3LVLW54bva3t4SzXcWKMdUWHEZR3V2VNKbmQgE8"),
+			(6, "FFFF3gBSSDFSvK2HBq4qgLH75DHqXWPHeCnR1BSksAMacBs"),
+			(5, "G7YVCdxZb8JLpAm9WMnJdNuojNT84AzU62zmvx5P1FMNtg2"),
+			(5, "15G1iXDLgFyfnJ51FKq1ts44TduMyUtekvzQi9my4hgYt2hs"), // proof https://kusama.subscan.io/extrinsic/16917610-2
+			(5, "Dcm1BqR4N7nHuV43TXdET7pNibt1Nzm42FggPHpxKRven53"),
+			(5, "J9nD3s7zssCX7bion1xctAF6xcVexcpy2uwy4jTm9JL8yuK"),
+			(4, "EGVQCe73TpFyAZx5uKfE1222XfkT3BSKozjgcqzLBnc5eYo"),
+			(4, "1eTPAR2TuqLyidmPT9rMmuycHVm9s9czu78sePqg2KHMDrE"), // proof https://kusama.subscan.io/extrinsic/16921712-3
+			(4, "14DsLzVyTUTDMm2eP3czwPbH53KgqnQRp3CJJZS9GR7yxGDP"), // proof https://kusama.subscan.io/extrinsic/16917519-2
+			(3, "13aYUFHB3umoPoxBEAHSv451iR3RpsNi3t5yBZjX2trCtTp6"), // proof https://kusama.subscan.io/extrinsic/16917832-3
+			(3, "H25aCspunTUqAt4D1gC776vKZ8FX3MvQJ3Jde6qDXPQaFxk"),
+			(3, "GtLQoW4ZqcjExMPq6qB22bYc6NaX1yMzRuGWpSRiHqnzRb9"),
+			(3, "15db5ksZgmhWE9U8MDq4wLKUdFivLVBybztWV8nmaJvv3NU1"), // proof https://kusama.subscan.io/extrinsic/16876631-2
+			(3, "HfFpz4QUxfbocHudf8UU7cMgHqkHpf855Me5X846PZAsAYE"),
+			(3, "14ShUZUYUR35RBZW6uVVt1zXDxmSQddkeDdXf1JkMA6P721N"), // proof https://kusama.subscan.io/extrinsic/16918890-8
+			(3, "CzEPpMr7XNS6dK7nQFnQbfnJQYLq7nvULK5kL9U8Zb6CTJm"),
```

Comment:
```suggestion
			(3, "12YzxR5TvGzfMVZNnhAJ5Hwi5zExpRWMKv2MuMwZTrddvgoi"),
```
Change for my polkadot address: 12YzxR5TvGzfMVZNnhAJ5Hwi5zExpRWMKv2MuMwZTrddvgoi 
Proof: https://kusama.subscan.io/extrinsic/16924324-3